### ISSUE TITLE
Docker scout is not beeing found ater running the install script in azure pipelines

### ISSUE THREAD

**Issue by achainarong** [docker/scout-cli#40]
Hey guys,

i followed your guide on the docker website. But when trying to run docker scout i get the error "docker 'scout' is not a docker command". And when i try to use the downloaded exectuable ~/.docker/cli-plugins/docker-scout i get the message that i should login into Dockerhub first even though i tried it over the pipeline with the command docker login and with the serviceconnection from azure itself.

**Comment by sc-rpt** [comment]
+ 1 Same issue following the installion steps provided (https://docs.docker.com/scout/integrations/ci/azure/) to use with azure pipelines

**Comment by alex-arana** [comment]
I had the same issue but used the following workaround:

```yaml
          - task: CmdLine@2
            displayName: Find CVEs on image
            env:
              INSTALL_DIR: $(Pipeline.Workspace)/cli-plugins
            inputs:
              script: |
                # Install the Docker Scout CLI
                curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s -- -b ${INSTALL_DIR}
                # Get a CVE report for the built image and fail the pipeline when critical or high CVEs are detected
                DOCKER_SCOUT_HUB_USER=$(DOCKER_HUB_USER) DOCKER_SCOUT_HUB_PASSWORD=$(DOCKER_HUB_PAT) $(INSTALL_DIR)/docker-scout cves $(image):$(tag) --exit-code --only-severity critical,high
```

**Comment by salvorusso** [comment]
Another solution could be using absolute path:
```yml
- task: CmdLine@2
  displayName: Install Docker Scout
  inputs:
    script: |
      # Install the Docker Scout CLI
      curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s --

      /home/vsts/.docker/cli-plugins/docker-scout version
```

However, installation steps provided for azure pipeline do not work!

**Comment by dloperab** [comment]
Thanks @salvorusso, your solution worked for me.

I did a little change:

```
  - task: CmdLine@2
    inputs:
      script: |
        # Install Docker Scout CLI
        curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s --
        export PATH=$PATH:/home/vsts/.docker/cli-plugins
        
        # Verify version
        docker-scout version
    displayName: "Install Docker Scout"
```

**Comment by paul-ri** [comment]
tldr; Specify the installation directory the pipeline needs by using the `DOCKER_CONFIG` env var:
```bash
curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh -o install-scout.sh
DOCKER_HOME=${DOCKER_CONFIG} sh install-scout.sh
# or
# sh install-scout.sh -b "${DOCKER_CONFIG}/cli-plugins"
```

----

I was always able to use `docker scout` after installing it in a pipeline, until I started using the `Docker@2` task with the 'login' command. Then I couldn't.

Turns out this task changes the DOCKER_CONFIG environment variable, which is used by docker to locate configuration files ([doc](https://docs.docker.com/reference/cli/docker/#environment-variables)).

I believe [this line of the Docker task](https://github.com/microsoft/azure-pipelines-tasks/blob/def5bfd5bddb75670dadd2deaf963d07067c80ba/Tasks/DockerV2/dockerlogin.ts#L8C16-L8C42) is the culprit: `connection.setDockerConfigEnvVariable();`

Printing the `DOCKER_CONFIG` env var on the console shows it's not empty anymore after that Docker@2 step, for example: `/home/vsts/work/_temp/DockerConfig_1754688106121`.

Suggestion: What about changing the [docker scout installation script](https://github.com/docker/scout-cli/blob/ff58efb9d2d249bdba20f241a6b25c94665bda97/install.sh#L15) to use DOCKER_CONFIG? 
```diff
-DOCKER_HOME=${DOCKER_HOME:-~/.docker}
+DOCKER_HOME=${DOCKER_CONFIG:-~/.docker}
```
